mifosng-android/src/main/java/com/mifos/mifosxdroid/SplashScreenActivity.java

```diff
@@ -50,11 +51,10 @@ private  void startappintro(final Context context) {
                 @Override
             public void run() {
                 //  Initialize SharedPreferences
-                SharedPreferences PREF = PreferenceManager
-                        .getDefaultSharedPreferences(context);
+                mprefManager = PrefManager.getPreferences();
 
                 //  Create a new boolean and preference and set it to true
-                misFirstStart = PREF.getBoolean("firstStart", true);
+                misFirstStart = mprefManager.getBoolean("firstStart", true);
```

Comment:
We have predefined methods named `putBoolean`  and `getBoolean` in `PrefManager.java` for saving and retrieving boolean from preferences. Please those methods.